### ISSUE TITLE
Automatic management of PostgreSQL for local development

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1962]
It would be nice if Liberapay had scripts to:

- automatically find or install a supported PostgreSQL version, at least on commonly used Unix-like operating systems
- configure and launch a dedicated instance for local development

This issue replaces #1304.

**Comment by techtonik** [comment]
Does kubernetes count as a solution?

**Comment by Changaco** [comment]
Kubernetes is for cluster management, it's irrelevant to this issue.

**Comment by techtonik** [comment]
@Changaco not really. You need to organize the cluster yourself, and then k8s takes over managing containers that are run there. It is an overkill for just two containers, of course.
